### PR TITLE
significant gameplay improvements. introduce a gitignore which the re…

### PR DESCRIPTION
**Pull request by logan-5**
…pository owner blatantly forgot

### CODE REVIEW

File: FarmNode.swift [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Foundation
 import SpriteKit
 
 class FarmNode: SKNode {
-  static let startingHealth = 3
+  static let startingHealth = 10
```
Comment (logan-5) [inline FarmNode.swift:5]: this felt lie a more appropritate gameplay decisionsmaking valuechoice

Comment (rsnively) [inline FarmNode.swift]: wow...

someone's generous

File: NinjaAttack/GameScene.swift [hunk 2]
```diff
@@ -106,16 +115,18 @@ class GameScene: SKScene, SKPhysicsContactDelegate {
...
+		monster.pickYourFarmAndGoThere(farms: self.livingFarms)
     
-    let speed = random(min: CGFloat(2.0), max: CGFloat(10.0))
-    let actionMove = SKAction.move(to: CGPoint(x: -monster.size.width/2, y: y),
-                                   duration: TimeInterval(speed))
-    monster.runAndRemove(actionMove)
+//    let speed = random(min: CGFloat(2.0), max: CGFloat(10.0))
+//    let actionMove = SKAction.move(to: CGPoint(x: -monster.size.width/2, y: y),
+//                                   duration: TimeInterval(speed))
+//    monster.runAndRemove(actionMove)
```
Comment (logan-5) [inline NinjaAttack/GameScene.swift:129]: :/

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+*.xcuserdatad
```
Comment (logan-5) [inline .gitignore:1]: nice!!

File: MonsterNode.swift [hunk 4]
```diff
@@ -3,9 +3,26 @@ import SpriteKit
...
-//    farm = farms[Int(random(min: 0.0, max: CGFloat(farms.count - 1)))]
+  func pickYourFarmAndGoThere(farms:[FarmNode]) {
+		if let farm = self.farm {
+			if farms.contains(farm) { return }
+		}
+		guard farms.count > 0 && !self.hasHitAFarm else { return }
+
+		if self.farm != nil {
+			print("breakpoint!")
+		}
```
Comment (rsnively) [inline MonsterNode.swift:17]: We don't need this any more do we?

**Review by rsnively — APPROVED** [review]

**Pull request merged**